eachdist.ini

```diff
@@ -43,7 +43,7 @@ packages=
     tests
 
 [experimental]
-version=1.10a0
```

Comment:
This version does not need to be bumped as we're not releasing new experimental packages at this time.